lib/cli/stats.js

```diff
@@ -220,19 +243,30 @@ async function getTemplatesSummary() {
   );
   summary.exportFiles.total = exportFilesNonEmpty.length;
   summary.exportFiles.externallyManaged = exportFilesExtMan.length;
+  summary.exportFiles.externallyManagedPerc = percentageRoundTwo(
+    summary.exportFiles.externallyManaged,
+    summary.exportFiles.total
+  );
 
   // Account Templates
-  const accountTemplatesNonEmpty = await listNonEmptyTemplates(
-    "accountTemplate"
-  );
+  const accountTemplatesNonEmpty =
```

Comment:
[SUGGESTION] Could further reduce to 1 line?

Reply:
Same as https://github.com/silverfin/silverfin-cli/pull/144#discussion_r1818930488